### ISSUE TITLE
Next Version / Release

### ISSUE THREAD

**Issue by p0wl** [ably/ably-asset-tracking-js#126]
Hey, we are integrating Ably Asset Tracking in our iOS and WebApp right now and are using this repository for the subscribers.

The latest release ([beta4](https://github.com/ably/ably-asset-tracking-js/releases/tag/v1.0.0-beta.4)) is from over a year ago and the API of Subscriber is changing significantly. Do you have any idea when the next version will be released?

With the old Subscriber API (in beta4) there is one subscriber per asset/tracker, but with the new API on the main branch, this changes, so each subscriber can handle multiple assets. With beta4 we ran into the max number of parallel connections, although only one tracker was online, because we were subscribing to ~50 trackers. This would only cause one connection per subscriber in the new api, right?

basically in beta4 the number of connections equals the number of tracked assets * number of clients, whereas on main, number of connections equals number of clients, correct?

Thanks for this library, it's exciting to work with it! We integrated the LocationAnimator successfully with our existing Mapbox map even on beta4, looks great!

**Comment by owenpearson** [comment]
Hey @p0wl, thanks for raising this and great to hear that you're enjoying working with the library!

You are right, the next version of the library will allow you to track multiple assets on a single client. Apologies that we haven't cut a release for it yet, we will bump the priority for this and I'll let you know in this issue as soon as it's available 🙂

**Comment by p0wl** [comment]
Thanks for the fast answer! Looking forward!

**Comment by owenpearson** [comment]
Hey @p0wl, I've just cut the release for v1.0.0-beta.5. As mentioned, the public API has changed quite a bit - please take a look at the README for usage examples for the new version and let me know if you run into any issues 🙂

**Comment by p0wl** [comment]
Hey @owenpearson , thank you! I've updated our integration and it works just fine!

The readme says the asset.start should be called with `trackingId` parameter, that is not needed: https://github.com/ably/ably-asset-tracking-js/blob/e748d363fe2216eca09cdb6074620b2f221862de/README.md?plain=1#L73

And can you also publish the release in GitHub? It's not listed here: https://github.com/ably/ably-asset-tracking-js/releases (but installing via NPM works fine already)

Have a great week!

**Comment by owenpearson** [comment]
Hey @p0wl, thanks for the suggestions! My bad for not publishing the release on GitHub, I've done that now and fixed the usage example in the README 🙂
